src/data/sql-quiz.ts

```diff
@@ -439,6 +441,17 @@ const sqlQuiz = [
       "The ROLLBACK TRANSACTION command will erase all modifications made to the data. Changes made to table and local variables will not be affected by this command.",
     Link: "https://learn.microsoft.com/en-us/sql/t-sql/language-elements/rollback-transaction-transact-sql?view=sql-server-ver16",
   },
+  {
+    Question:
+      "Which of the following is not a distinct relationship in a SQL database",
+    Answer: "All-to-one",
+    Distractor1: "One-to-one",
+    Distractor2: "One-to-many",
+    Distractor3: "Many-to-many",
+    Explanation:
+      "Database data can either be linked one-to-one where one record is related to only one record in another table ,one-to-many where one record corresponds to many records in another table, or many-to-many where many records correspond to many records in another table. ",
```

Comment:
Let's shorten this a bit because remember that if the user wants to learn more they can always click on the learn more link to read the article 👍 

```suggestion
      "Database data can either be linked by one-to-one, one-to-many, or many-to-many relationships.",
```